src/cmd/health.js

```diff
@@ -0,0 +1,113 @@
+/* Copyright (C) 2020 NooBaa */
+'use strict';
+
+const os_util = require('../util/os_utils');
+const { make_https_request } = require('../util/http_utils');
+const minimist = require('minimist');
+
+
+
+const HELP = `
+Help:
+
+    "nsfs" is a noobaa-core command runs a local S3 endpoint on top of a filesystem.
+    Each sub directory of the root filesystem represents an S3 bucket.
+    Health command will return the health status of deployed nsfs.
+`;
+
+const USAGE = `
+Usage:
+
+    node src/cmd/health [options...]
+`;
+
+
+const OPTIONS = `
+Options:
+
+    --nsfs_type <type>         (default nc)   Set the nsfs type for heath check.
+
+`;
+
+function print_usage() {
+    console.warn(HELP);
+    console.warn(USAGE.trimStart());
+    console.warn(OPTIONS.trimStart());
+    process.exit(1);
+}
+
+const HOSTNAME = "localhost";
+const PORT = 6443;
+const SERVICE = "nsfs";
+
+async function nc_nsfs_health() {
+    const service_status = await os_util.exec('systemctl show -p ActiveState --value ' + SERVICE, {
```

Comment:
I would split the function's code to multiple small functions get_memory_status() etc